config/config.go

```diff
@@ -136,6 +135,10 @@ func New(fileName string) (config *Config, err error) {
 		config.Monitoring.WorkingDir = config.StorageDir
 	}
 
+	if config.ServiceTTLDays == 0 {
```

Comment:
you could use the previous approach to set data in config structure before unmarshall

Reply:
Done